src/main/java/com/capitalone/dashboard/webhook/github/GitHubPullRequestV3.java

```diff
@@ -141,6 +141,10 @@ public String process(JSONObject prJsonObject) throws MalformedURLException, Hyg
         Object base = restClient.getAsObject(pullRequestObject, "base");
         String branch = restClient.getString(base, "ref");
 
+        if(!isRegistered(repoUrl, branch)) {
+            return "Repo: <" + repoUrl + "> Branch: <" + branch + "> is not registered in Hygieia";
```

Comment:
@chzhanpeng  instead of break, how about set enabled and pushed flag to false and accept payload ? That way we can speedup collector by not pulling these repo/branches.